vouches/Cargo.toml

```diff
@@ -0,0 +1,51 @@
+[package]
+name = "pallet-encointer-vouches"
+version = "1.2.0"
+authors = ["encointer.org <alain@encointer.org> and Parity Technologies <admin@parity.io>"]
+edition = "2021"
+
+[dependencies]
+approx = { version = "0.5.1", optional = true }
+codec = { package = "parity-scale-codec", version = "3.6.1", default-features = false, features = ["derive"] }
+log = { version = "0.4.14", default-features = false }
+scale-info = { version = "2.5.0", default-features = false }
+
+# local deps
+encointer-primitives = { path = "../primitives", default-features = false, features = ["serde_derive"] }
+
+# substrate deps
+frame-benchmarking = { version = "4.0.0-dev", default-features = false, optional = true, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v1.0.0" }
+frame-support = { version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v1.0.0" }
+frame-system = { version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v1.0.0" }
+pallet-timestamp = { version = "4.0.0-dev", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v1.0.0" }
+sp-core = { version = "21.0.0", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v1.0.0" }
+sp-runtime = { version = "24.0.0", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v1.0.0" }
+sp-std = { version = "8.0.0", default-features = false, git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v1.0.0" }
+
+[dev-dependencies]
+approx = "0.5.1"
+sp-io = { version = "23.0.0", git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v1.0.0" }
+test-utils = { path = "../test-utils" }
+
+[features]
+default = ["std"]
+std = [
+    "codec/std",
+    "log/std",
+    "scale-info/std",
+    "approx/std",
+    # local deps
+    "encointer-primitives/std",
+    # substrate deps
+    "frame-support/std",
+    "frame-system/std",
+    "sp-std/std",
```

Comment:
I was wondering why this works, but I see now that you don't compile the benchmarks for this pallet with the tests. I think we need a `frame-benchmarking?/std` here.